tests/test_mla_page.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+import torch
+
+import flashinfer
+
+
+def test_append_mla_paged_kv_cache():
```

Comment:
Can you design some stronger test cases, I apologize that the previous `test_page.py` is very weak (most of the tests around append page APIs is performed in C++ unittests and in the future we should move them to python).

By "stronger" I mean more cases (nnz/append length), data types, page_size etc.